pipeline/api-service/base/config-map.yaml

```diff
@@ -11,16 +11,19 @@ data:
         "DBName": "mlpipeline",
         "GroupConcatMaxLen": "4194304"
       },
-      "ObjectStoreConfig":{
+      "ObjectStoreConfig": {
         "AccessKey": "minio",
         "SecretAccessKey": "minio123",
         "BucketName": "mlpipeline",
+        "PipelinePath": "pipelines",
         "Secure": false
       },
       "InitConnectionTimeout": "6m",
       "DefaultPipelineRunnerServiceAccount": "pipeline-runner",
-      "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST": "ml-pipeline-ml-pipeline-visualizationserver",
-      "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT": 8888
+      "VisualizationService":{
+        "Name": "ml-pipeline-visualizationserver",
+        "Port": 8888
+      }
```

Comment:
It seems like the apiserver still complains about `ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST` and `ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT` (KFP version 0.5.1).

Is there a change for the `VisualizationSerice.{Name,Port}`? I can't find anything related to that.
I believe this is related to the constants here: https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/client_manager.go

Reply:
Can you file a bug about that? Does apiserver break without visualization server service host/port?

In multi user mode, the new env vars are used, introduced in https://github.com/kubeflow/pipelines/pull/3495/files#diff-95658ae516fd293dd7cb039e484b558dR20. (because visualization server will live in user namespaces)